rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    @person.rentals << self
+    @book = book
+    @book.rentals << self
```

Comment:
Great job on implementing the ` belongs_to` relationship between `book`, `person` and `rental` :+1: